src/components/Register.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import {InputGroup, Form, FormControl, Button, Alert} from 'react-bootstrap';
+
+import {fetchUser, getToken, setToken as setLocalToken} from '../util'
+
+
+export default props => {
+  const {setUser, setToken, token} = props;
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+  const [error, setError] = useState();
+    
+  useEffect(() => {
```

Comment:
This is getting the localstorage token and setting it on state?  Probably define this in the App component?

Reply:
Also, remove the `getToken` function call, in favor of `getLocalToken`?